File: returned/python/today21/kth_smallest_element_in_a_bst_test.py
from kth_smallest_element_in_a_bst import Solution, TreeNode

def test1():
    n1 = TreeNode(1)
    n2 = TreeNode(2)
    n3 = TreeNode(3)
    n4 = TreeNode(4)
    n3.left = n1
    n3.right = n4
    n1.right = n2
    assert Solution().kthSmallest(n3, 1) == 1

File: csharp/LeetTests/Today4/KthSmallestElementInABSTTest.cs
using Today4.KthSmallestElementInABST;
namespace Today4;

internal class KthSmallestElementInABSTTest
{
    [Test]
    public void Test1()
    {
        var solution = new Solution();
        var n3 = new TreeNode(3);
        var n1 = new TreeNode(1);
        var n4 = new TreeNode(4);
        var n2 = new TreeNode(2);
        n3.left = n1;
        n3.right = n4;
        n1.right = n2;
        var result = solution.KthSmallest(n3, 1);
        Assert.That(result, Is.EqualTo(1));
    }
}